src/modules/cluster/cluster-list/cluster-list.html

```diff
@@ -58,15 +58,15 @@ <h1 class="bold-text">Clusters</h1>
     </center>
     <div class="list-group list-view-pf list-view-pf-equalized-column" ng-repeat="cluster in clusterCntrl.filteredClusteredList = (clusterCntrl.clusterList | filter: clusterCntrl.searchBy | orderBy: clusterCntrl.ascOrder + clusterCntrl.orderBy)">
         <div class="list-group-item">
-            <div class="list-view-pf-left text-center">
+            <div class="list-view-pf-left text-center col-md-1">
```

Comment:
still you are not using all 12 col division . can you try with all 12 column division i think that will best fit in UI.